portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,172 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Andrew Rubinstein</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body id="body">
+      <div style="padding: 7px 25px;">
+       <a class="button" href="https://github.com/andrewrubinstein">
+       <h3>Github </h3>
+       </a>
+    <a class="button" href="https://www.linkedin.com/in/andrew-rubinstein-a817b41a9/">
+    <h3>LinkedIn</h3>
+    </a>
+    </div>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <div id="subcontent">
+    <h1 style="text-align: center">Andrew Rubinstein</h1>
+      <h2>An Introduction to me</h2>
+      <p>
+          “Success consists of moving from failure to failure without a loss of enthusiasm.”  <br>
+          -Often attributed to Winston Churchill <br>
+          This is a quote that continues to help me to this day.<br><br>
+          I Spent my early 20's working for a software company that developed a system that did inventory,<br>
+          management, Accounting, and some customer relationship management or CRM tasks. <br>
+          I took a break from formal education to pursue an opportunity.<br><br>
+          The company gave me responsibilities that included sequenced query language (SQL) development, sales, and implementation of their software.  <br><br>
```

Comment:
Nit: SQL stands for _structured_ query language although writing out the abbreviation is uncommon.